jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenProjectPropertiesTest.java

```diff
@@ -776,351 +706,346 @@ public void testGetChildValue_nullValue() {
 
   @Test
   public void testGetSpringBootRepackageConfiguration_pluginNotApplied() {
-    Assert.assertEquals(
-        Optional.empty(), mavenProjectProperties.getSpringBootRepackageConfiguration());
+    assertThat(mavenProjectProperties.getSpringBootRepackageConfiguration())
+        .isEqualTo(Optional.empty());
   }
 
   @Test
   public void testGetSpringBootRepackageConfiguration_noConfigurationBlock() {
-    Mockito.when(mockMavenProject.getPlugin("org.springframework.boot:spring-boot-maven-plugin"))
+    when(mockMavenProject.getPlugin("org.springframework.boot:spring-boot-maven-plugin"))
         .thenReturn(mockPlugin);
-    Mockito.when(mockPlugin.getExecutions()).thenReturn(Arrays.asList(mockPluginExecution));
-    Mockito.when(mockPluginExecution.getGoals()).thenReturn(Arrays.asList("repackage"));
-    Mockito.when(mockPluginExecution.getConfiguration()).thenReturn(null);
-    Assert.assertEquals(
-        Optional.of(new Xpp3Dom("configuration")),
-        mavenProjectProperties.getSpringBootRepackageConfiguration());
+    when(mockPlugin.getExecutions()).thenReturn(Arrays.asList(mockPluginExecution));
+    when(mockPluginExecution.getGoals()).thenReturn(Arrays.asList("repackage"));
+    when(mockPluginExecution.getConfiguration()).thenReturn(null);
+    assertThat(mavenProjectProperties.getSpringBootRepackageConfiguration())
+        .isEqualTo(Optional.of(new Xpp3Dom("configuration")));
   }
 
   @Test
   public void testGetSpringBootRepackageConfiguration_noExecutions() {
-    Mockito.when(mockMavenProject.getPlugin("org.springframework.boot:spring-boot-maven-plugin"))
+    when(mockMavenProject.getPlugin("org.springframework.boot:spring-boot-maven-plugin"))
         .thenReturn(mockPlugin);
-    Mockito.when(mockPlugin.getExecutions()).thenReturn(Collections.emptyList());
-    Assert.assertEquals(
-        Optional.empty(), mavenProjectProperties.getSpringBootRepackageConfiguration());
+    when(mockPlugin.getExecutions()).thenReturn(Collections.emptyList());
+    assertThat(mavenProjectProperties.getSpringBootRepackageConfiguration())
+        .isEqualTo(Optional.empty());
```

Comment:
```suggestion
    assertThat(mavenProjectProperties.getSpringBootRepackageConfiguration()).isEmpty();
```

Reply:
Oh, similar to the `Path` issue above, I realized `isEmpty()` is available only in `Truth8.assertThat()`. Maybe I should use `Truth8.assertThat()` explicitly?

Reply:
BTW, for other normal assertions, `Truth.assertThat` doesn't compile, so it looks like I need to use both `Truth.assertThat` and `Truth8.assertThat`?